timbermill-client/src/main/java/com/datorama/timbermill/ElasticsearchClient.java

```diff
@@ -124,83 +130,103 @@ private void deleteOldIndex(String currIndex) throws IOException {
         if (daysBackToDelete < 1){
             return;
         }
-        String indexPrefix = currIndex.split("-")[0];
-        String oldIndex = getTaskIndexWithEnv(indexPrefix, ZonedDateTime.now().minusDays(daysBackToDelete));
-        GetIndexRequest existsRequest = new GetIndexRequest(oldIndex);
+        String[] split = currIndex.split("-");
+        String indexPrefix = split[0];
```

Comment:
1. lets put all delimiters such "-" as consts
2. seems a little risky accessing the array like that, maybe add some protection